Dashboard/Client/SessionManagement/ClientSessionManager.cs

```diff
@@ -50,12 +55,13 @@ public ClientSessionManager()
         /// </param>
         public ClientSessionManager(ICommunicator communicator)
         {
+            TraceManager session = new();
             moduleIdentifier = "Dashboard";
             _serializer = new Serializer();
             _communicator = communicator;
             _communicator.Subscribe(moduleIdentifier, this);
-            TraceManager session = new();
-            session.TraceListener();
+            clientBoardStateManager = ClientBoardStateManager.Instance;

```

Comment:
Why use the actual whiteboard for testing. 
u can remove this whiteboard state manager here.

Reply:
That I am not using in testing

Reply:
When I will use, I will create a fake for it